### DISCUSSION TITLE
Leveraging Sunbird capabilities (existing/ proposed) to further Learning Outcomes

### DISCUSSION THREAD

**Discussion by rhwarrier** [Sunbird-Ed/Community#502] [General Discussion]
The Sunbird suite has been leveraged by several adopters including [DIKSHA](https://diksha.gov.in), [iGot Karmayogi](https://igotkarmayogi.gov.in) etc. to leverage the capabilities of the digital medium to deliver learning content to users. This has been an effective way to creating learning opportunities for the learners on the platform

The next stage in this journey is to progress from opportunities to outcomes.
Learning outcomes are the end goal that need to be achieved, and this post is intended to be an anchor for discussions around furthering the capability of learners to achieve their intended outcomes. 

Please share ideas and thoughts on how the Sunbird suite can be used to further learning outcomes.
This can be in the context of existing SB capabilities, or suggestions in terms of new contributions that can help with this cause.

Let's use this thread to discuss and arrive at a actionable set of ideas that can help with the goal of achieving learning outcomes effectively.

@Krishnaj20  @surendrasinghs  @jagadishbabu

**Comment by arkamaini** [comment]
what we have realised from our Genie days, when it comes to learning opportunities/outcomes , we need to need '**boost**' the community to create content that can target learning outcomes. With the advent of AI tools like MidJourney, DALL-E and ChatGPT content creation has become easier that ever before. Everything that we struggled with when designing our Content Authoring Tool can be solved easily with AI powered text and media creation. Can we create a new building block that can enable content-creation with AI powered text and media generator, which is fine tuned for learning content? Imagine motivated teachers who has ideas without the technical expertise creating 'hyper-contextualised' content for their classroom with a simple text prompt.

**Comment by Jayaprakash8887** [comment]
As we learnt from the meeting, there are 2 areas that can be focused to enable Learning Outcome related functionalities.
            a. Features focused on learning process.
            b. Features focused on assessing the learning.

As part of the user's profile, we already capture the User's state (in terms of Learning framework - BMGS user is primarily focused on) in DIKSHA. We can also capture **preferred learning languages** from the user apart from current academic learning framework. This can be used for suggesting contents.

Based on user's learning framework, things I believe might help a user improve his learning are

            PROCESS RELATED FEATURES:
                  a. Tracking the learning path of user in textbooks and suggesting most liked/viewed/downloaded contents (Not linked to current Textbook) around the topics of the textbook user is learning. ("Help me" button in textbook where user can lookout for different type of contents based on topics in textbook).
                  b. Popping up quiz question/question set/fun facts or sorts, based on learning path once in few minutes. (Ability to switch on and off this feature by a user to be there).
                  

            ASSESSMENT RELATED FEATURES:
                  a. Ability for user to be able to generate own assessments based on topics. Reverting with the suggestions to the learner to improve focus on topics where shortcomings are found.

**Comment by Krishnaj20** [comment]
'Content' will play a major role in the journey from 'access' to 'learning'. Although there are many learning apps which provide personalized learning solutions to learners there is no app which provides learners the choice to choose the content that suits them for a specific topic. This is largely due to the fact that content creation is a costly activity - both in terms of time taken and the availability of quality creators - and hence the inability of organizations to employ multiple creators to create a variety of content on the same topic. Further, majority of this content is propriety of the organizations and is not made available to everyone. 

Can AI be leveraged to assist creators to create variety of content in Indic languages in a fraction of time and cost? This content can be used by multiple learning apps to enable the journey from 'access' to 'learning'. To start with, we can focus on limited scope (for ex., class 5, math, algebra) but the reach can be at the level of a state.

**Comment by gandham-santhosh** [comment]
Here are my thoughts to improve the learning outcomes 
- Integrating with the external learning management systems and open digital libraries to enhance the quality of content and improving the learning capabilities of the user
- Providing gap analysis insights and growth path to the user based on the learning journey and assessments that he/she went through (telemetry data)
- Include gamification elements like badges, points, leaderboards etc to engage and motivate the user
- Getting feedback from the learner especially the benefits he/she gained through the course/content. This information might help to advice/suggest the similar content/courses to learners of same category and improve the learning journey

**Comment by chitranshu-keshav** [comment]
Below are my thoughts to enhance the learning outcome of users using a two-step approach:

Step-1: Streamline Content Discovery. Make content discovery as effortless as possible for users, ensuring they can find useful, relevant and personalized content with minimal steps and clicks
Step-2: Effective User Engagement. After content discovery, engage the user in a way that encourages them to consume additional related content, thereby enhancing their learning outcome.

Enhancements for Step-1: 
1) Replace BMGS filters, persona, and location entries with a text-based search function that leverages the user's profile for content discovery. The user's profile can be automatically generated based on their interactions and searches on the site. This approach will eliminate the need for users to manually enter multiple data points, making content discovery easier, more efficient, relevant and personalized.
2) Improve the intelligence of QR code searches by utilizing the QR code data, such as the board, class, subject, and lesson. This information can be used to present additional relevant content to the user as well as removing Irrelevant content, such as displaying the textbook when scanning a QR code for a specific lesson of that textbook. The QR code data also can be  used to build/enhance the profile and present personalized contents to user  

Enhancements for Step-2:
1) Enhance the question-and-answer bot to prompt users to explore related topics after providing an answer to a specific question. For example, if the user asks about the Himalayas, the bot can suggest learning about other mountain ranges in India. This will encourage users to delve deeper into connected subjects, thereby enhancing their learning outcomes.
2) Integrate existing bots into the web pages and make them more prominent for users. This ensures that users are aware of the bot's presence and can readily engage with it. The bots can be activated during specific actions, such as after displaying search results, to gather feedback on content relevance. Additionally, the bots can engage users upon completion of a piece of content, encouraging them to explore further topics related to their interests and learning goals.

**Comment by rhwarrier** [comment]
There seem to be several common themes bubbling up - including leveraging AI for effective content, as well as linking to existing libraries/ systems and providing a personalised learning journey.

Here are some thoughts on leveraging some of the existent capabilities as well:

DATA :
Leverage existing data capabilities at a granular (content) level to create ‘stories’ that can create hooks and further engagement.
For e.g.:
> Have you tried this popular course? 87% of learners have taken this course successfully.
> Congratulations on completing the assessment - your high score puts you in the top 1% of learners who have taken this assessment
> You are a quick learner - you completed the course faster than 93% of other learners

COLLABORATION :
> Enable learner collaboration on existing/ popular channels - for example, Wordpress has a plugin that allows a group chat to be added on a Wordpress website. The means of collaborating remains the same - it just happens in a specific context now, i.e. one of learning.

ENGAGEMENT :
> A little out-of-the-box, but is there a way to create a new type of content that is in line with modern trends and tech, that will excite learners enough to keep coming back?
For example, users keep going back to FB/ Instagram just to see the new posts. Likewise, is there a new bite-sized type of content we can try out? Memes are hugely popular with the student/ teen audience - can learning content be structured in a similar fashion, for instance? 

SCHEDULING :
Integrate with Google calendar, allowing learners to ‘schedule’ their classes - plan what they want to learn, and when. Also use calendar notifications to remind/ prompt learners to finish their planned work. This will ride the existing user behaviour of being interrupt-led. 

UPLOAD/ SHARE NOTES :
Allow learners to upload & share their notes via the platform. This is also existing  user behaviour which is a good candidate for the platform. Users whose notes are good quality or widely shared can receive recognition across the platform. This can also tie back to the daily work that learners undertake, and give them reason to access the platform. It also acts as a means of organic content creation - Users who consistently produce good notes can be incentivised to become content creators for the platform.

**Comment by PoojaJManjooran** [comment]
For achieving learning outcomes, we should be able to help accelerate the learning journey. This necessarily means improving the efficiency of the learning process. Some key focus areas with respect to the above could be :

1. Enabling the learners to be able make the best out of the time spent. One of the possible ways of achieving this is by capsuling the  learning material into whatever is most relevant for each of the user personas. For eg, If there is a summary of the entire Topic/section covering the most important and relevant information alone. This will also help learners to be prepared before starting the detailed learning and also for revision. We can have different versions of the same content varying in the levels of detailing.

2. For learners preparing for examination, showing potential questions or applications that could come from each of the sections/sub-topics will also be helpful for learners to prepare for answering the questions from whatever they are learning. ie When a learner is going through a content, the potential questions or applications can be shown side-by-side or on-demand. In this way, helping the learners to be prepared, as and when they are going through the content.

We could auto-generate both of the above features

**Comment by sajeshkayyath** [comment]
**Context**
A learner comes to the portal to learn a particular subject. He reads / listens through the passages. As part of active learning he takes notes and recalls from memory.

Some part of the learning material can be just informational, while other parts holding key information. 

**Some possible learning boosters,**

- If one is preparing for an exam, wouldn't it be beneficial to the learner to get an idea which part of the learning material is important ones.
An indicator of some kind for the content that is to be must read. Probably derived out of the below, but not limited to,

  1. The importance of the topic
  2. The chances of recurrence of the questions from this topic from various competitive exams.
  3. The number of discussions around that topic

-  Wouldn't it be beneficial for the learner to see the past or possible questions that can come from a particular topic, without navigating through many pages. So the learner gets to see different flavours of the question and do the preparation.

-  Wouldn't it be beneficial for the learner to self create questions for the learning purpose and probably share it with others.

- Can I search for questions on a particular topic and then getting redirected to the detailed content.

 - What if the learner gets to see the student benchmarks from his class, i.e; how much on an average other students from his class is spending time on the learning portal, what topics are they learning. This can probably become a catalyst for the learners motivation to spend more time on learning.

- May be a badge of some kind for the active learners to show case

**In summary this document covers the below points,**
- A indicator for the content / topic that is critical for learning the subject
- Within the content, discovering the past questions / possible questions 
- Self creation of questions for learning purpose
- Searching through the questions and discovering the relevant content
- Getting to see the benchmark of a class (average time spent, tests completed, topics under learn etc)
- Recognition badges for active learning

**Comment by sreeharshaekstep** [comment]
Ideas to enhance learning journey:

1. Personalized Learning Paths: Develop a system (on DIKSHA) that uses student data to create personalized learning paths for each student for each academic year or semester or trimester. This would involve assessing the student's strengths and weaknesses based on assessments(conducted online/offline) data and providing tailored content and exercises to address their individual needs. Utilize AI to provide simple local-language-interaction via chatbot which can provide instant feedback and assessment to students. AI can analyze student responses, identify areas of improvement, and offer targeted feedback, enabling self-paced learning and immediate reinforcement.
2. Virtual Mentoring and Tutoring: Introduce virtual mentoring and tutoring programs on DIKSHA, where experienced educators or subject matter experts can provide one-on-one guidance and support to students who need additional assistance or enrichment.
3. Collaboration and Discussion Platforms: Integrate collaborative tools on DIKSHA that enable students to engage in online discussions, group projects, and peer-to-peer learning. This fosters collaboration, critical thinking, and communication skills among students.

All three can be linked to personalized learning paths. 

Ideas for giving agency to ecosystem:

1. Research and Innovation Hub: DIKSHA can establish itself as a research and innovation hub in the education sector. By partnering with academic institutions, research organizations, and think tanks, DIKSHA can drive research initiatives to explore innovative pedagogical approaches, leverage emerging technologies, and identify best practices to improve student learning.
2. Teacher Training and Certification Programs: Collaborate with teacher training institutes and certification bodies to offer online training programs and certification courses through its platform. This would provide teachers with professional development opportunities and help them enhance their teaching skills, ultimately benefiting student learning outcomes.

**Comment by mmanimegalai** [comment]
I found myself constantly pondering the question “what is learning?” for the last 3-4 days. In the process, few more questions started to creep in - Yes, we have helped millions of children, teachers and other users get access to content to help in their  learning journey. But are the users truly learning? Is something missing in the learning process? What are the gaps? How should it be filled? What are the ways in which true learning can be achieved? etc.,

I have not been able to get answers for all these questions. However i have put together some of my thoughts that i gathered so far in a deck. Have tried my best to keep my thoughts very crisp , so as to not lose focus. Please feel free to criticize or add anything that you think will add value

https://docs.google.com/presentation/d/1pwKFlbYE6wGgI-4oh1b_MNDpkk-mvU5TIUxFF8YEhdI/edit#slide=id.g24944135815_0_52

**Comment by mmanimegalai** [comment]
@rhwarrier Should this thread be like a banner (Announcement) to catch more attention?

**Comment by mmanimegalai** [comment]
I thought through some of the real core issues that learners face today (especially children) today and took a stab at how they can be solved using AI 

1.  **"A picture is worth a thousand words"** - Can AI be used to learn, remember and explore concepts, experiments, dates etc., ONLY through picture? be it a diagram or tree structure or flow diagram? 
Instead of long paragraphs/text, can the **response to a child's questions always be pictorially represented?** 
Eg: To understand how a Heart functions, it is easier for a child to understand through diagram with labels and functions that will stay long. Anything to do with the heart's function, the child will always co-relate with the diagram to recollect or corelate
Similarly say in physics all that a child needs to remember about the experiment to "demonstrate light travels in a straight line" is just a picture of cardboards with a hole and a candle and nothing else 
In History, dates and rulers can be chronologically arranged in a tree structure to understand and remember better

2. Ability to provide a short (really crisp) **summary** of every topic, chapter, concept learnt for easy reference. But again as much as possible only in the form of pictures

3. Ability to let the user self test their understanding after every concept learnt. Say after reading about a topic (eg: Photosynthesis), quick questions should get automatically generated for the user to take and reflect upon.

4. Improving Assessments - Instead of standard set of questions asked, based on how the user performs, more easier or complex additional questions should follow

5. Ability to translate content in local language for understanding. **It is not for better understanding, it is for basic understanding**. sometimes, it is absolutely nothing that a child understands when it is in a language that s/he is not well versed at .This is not an issue in rural areas alone. This is the case with children in cities too

6. Learner's interest should be identified based on how they interact with the app (how much time they spend, browsing patterns, how many time forward/rewind option was used, skip etc.,)
